crates/rspack/tests/fixtures.rs

```diff
@@ -40,9 +41,9 @@ fn tree_shaking(fixture_path: PathBuf) {
         options.optimization.used_exports = UsedExportsOption::True;
         options.builtins.tree_shaking = TreeShaking::False;
 
-        if options.optimization.side_effects.is_enable() {
```

Comment:
disable for now,  I will recover when after covering snapshot tests.

Reply:
temporarily disabled to reduce some noise when investigating diff compared with the previous implementation.